tests/page-objects/apdex/page.js

```diff
@@ -0,0 +1,243 @@
+const { browser, driver } = require('@wdio/globals');
+const { execSync } = require('child_process');
+
+module.exports = class Page {
+
+  async waitForDisplayedAndRetry(selector, retryTotal = 20, retryCount = 0) {
+    const TIME_OUT = 1000 * 60 * 20;
```

Comment:
@lorerod this has been left like this because some of the instances taake a very long time to display the contactList after loading the data - reducing the time makes some of the tests time-out